### DISCUSSION TITLE
Keyboard shortcut: Open next log entry

### DISCUSSION THREAD

**Discussion by jameshulse** [opcodesio/log-viewer#363] [Ideas]
I'd love a keyboard shortcut, maybe the arrow keys or 'n', that would open the next log entry. That way I can make my way back through the logs without clicking to close my current entry and clicking to open the next one, over and over. I would also expect the shortcut to close my current entry.

**Comment by arukompas** [comment]
hey @jameshulse , the feature is now available in **v3.10.0** :) use [n] or [p] for next and previous logs, or [j] / [k] if you're used to vim motions.

If in doubt, hit the question mark key on your keyboard to display the available keyboard shortcuts.

Hope this helps!
